apis/reaper/v1alpha1/reaper_types.go

```diff
@@ -172,6 +172,10 @@ type ReaperClusterTemplate struct {
 	// and "password".
 	// +optional
 	JmxUserSecretRef corev1.LocalObjectReference `json:"jmxUserSecretRef,omitempty"`
+
+	// Defines the secret which contains the shiro.ini file controlling the Reaper UI and REST API authentication.
```

Comment:
Nit: The comment states that the secret should contain the whole shiro.ini file, however it seems that in our tests, we expect the secret to contain just a `username` and a `password` entry, like other secrets in this project.